PATHFIND/SetAllowStreamHeistIslandNodes.md

```diff
@@ -0,0 +1,22 @@
+---
+ns: PATHFIND
+aliases: ["_SET_AI_GLOBAL_PATH_NODES_TYPE"]
+---
+## SET_ALLOW_STREAM_HEIST_ISLAND_NODES
+
+```c
+// 0xF74B1FFA4A15FBEA
+void SET_ALLOW_STREAM_HEIST_ISLAND_NODES(int type);
```

Comment:
That parameter is actually of `bool` type, you may name it `bAllow` I suppose.

Would change to the following:
`void SET_ALLOW_STREAM_HEIST_ISLAND_NODES(cs_type(int) BOOL bAllow)`